authorizations.md

```diff
@@ -54,6 +54,23 @@ Return codes:
 * 401 Unauthorized - if you are not authenticated and an eperson is specified
 * 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators and the user specified in the eperson parameter can access
 
+#### objects
+**/api/authz/authorizations/search/objects?uuid=<:uri>[&eperson=<:uuid>&feature=<:string>]**
```

Comment:
tiny typo here... it says `uuid=<:uri>` when it obviously should say `uuid=<:uuid>`.